### ISSUE TITLE
Error dialog title not displaying correctly

### ISSUE THREAD

**Issue by goneall** [OpenChain-Project/Online-Self-Certification-Web-App#111]
The changes for the error dialog still has issues.

Below are screenshots of what is displayed when an error occurs:
![image](https://user-images.githubusercontent.com/378172/52738028-95621d80-2f82-11e9-919e-717cdd21072f.png)

After clicking OK, the following appears (not the empty dialog with just a checkbox):

![image](https://user-images.githubusercontent.com/378172/52738084-b62a7300-2f82-11e9-9cf3-52cc47741a57.png)

This error occurs after the merge of PR #108

**Comment by goneall** [comment]
To duplicate the issue, you need to force an error.

I did this by modify the code CertificationServlet.java commenting out line 251 `gson.toJson(updateSurvey(tag, commit, locale, false), out);` and adding `throw(new Exception("Test error"));`

**Comment by goneall** [comment]
I tried forcing another error in a different screen by changing code in CertificationServlet.java around line 222 to:

```
else if (requestParam.equals(GET_SURVEY_RESPONSE)) {
	            	//gson.toJson(user.getSurveyResponse(locale), out);
	            	throw(new Exception("Test error"));
```

This produced the following result:
![image](https://user-images.githubusercontent.com/378172/52738426-7f089180-2f83-11e9-9792-c30c0f6fe9eb.png)

**Comment by goneall** [comment]
Resolved with PR #117
